src/main/java/org/jabref/model/search/rules/SearchRules.java

```diff
@@ -44,6 +44,6 @@ static SearchRule getSearchRule(EnumSet<SearchFlags> searchFlags) {
     }
 
     public enum SearchFlags {
-        CASE_SENSITIVE, REGULAR_EXPRESSION, FULLTEXT, KEEP_SEARCH_STRING
+        CASE_SENSITIVE, REGULAR_EXPRESSION, FULLTEXT, KEEP_SEARCH_STRING, FILTERING_SEARCH
```

Comment:
I don't think that this is correct. Filtering search is a mode, how the search results are displayed in the table, the search flags instead are directly connected to the search query. I believe this needs to be kept seperate.

Reply:
Done in: [`c62809f` (#11510)](https://github.com/JabRef/jabref/pull/11510/commits/c62809f5524101d84f36c5748ee8ba7741b75298)